cmd/cluster/log-exporter/query_log_exporter.go

```diff
@@ -68,13 +68,36 @@ var enableDbQueryLoggingCmd = &cobra.Command{
 
 		resp, r, err := authApi.EnableDbQueryLogging(clusterId).PgLogExporterConfigSpec(
 			ybmclient.PgLogExporterConfigSpec{ExportConfig: exportConfig, ExporterId: integrationId}).Execute()
+		dqlConfig := resp.GetData()
 
 		if err != nil {
 			logrus.Debugf("Full HTTP response: %v\n", r)
 			logrus.Fatalf(ybmAuthClient.GetApiErrorDetails(err))
 		}
 
-		formatter.DbQueryLoggingWriteFull(resp.GetData(), integrationName)
+		msg := fmt.Sprintf("The db query logging for cluster %s is being enabled", clusterName)
+		if viper.GetBool("wait") {
+			returnStatus, err := authApi.WaitForTaskCompletion(clusterId, ybmclient.ENTITYTYPEENUM_CLUSTER, ybmclient.TASKTYPEENUM_ENABLE_DATABASE_QUERY_LOGGING, []string{"FAILED", "SUCCEEDED"}, msg)
+			if err != nil {
+				logrus.Fatalf("error when getting task status: %s", err)
+			}
+			if returnStatus != "SUCCEEDED" {
+				logrus.Fatalf("Operation failed with error: %s", returnStatus)
+			}
+			fmt.Printf("DB query logging has been enabled for the cluster %v\n", formatter.Colorize(clusterName, formatter.GREEN_COLOR))
+
+			respC, r, err := authApi.GetDbLoggingConfig(clusterId).Execute()
```

Comment:
This code is also repetitive, you can create one more method to fetch the DB logging config.